docs/source/markdown/podman-create.1.md

```diff
@@ -859,7 +859,7 @@ Security Options
 
 - `unmask=ALL or /path/1:/path/2` : Paths to unmask separated by a colon. If set to **ALL**, it will
   unmask all the paths that are masked or made read only by default.
-  The default masked paths are **/proc/acpi, /proc/kcore, /proc/keys, /proc/latency_stats, /proc/sched_debug, /proc/scsi, /proc/timer_list, /proc/timer_stats, /sys/firmware, and /sys/fs/selinux.**  The default paths that are read only are **/proc/asound, /proc/bus, /proc/fs, /proc/irq, /proc/sys, /proc/sysrq-trigger**.
+  The default masked paths are **/proc/acpi, /proc/kcore, /proc/keys, /proc/latency_stats, /proc/sched_debug, /proc/scsi, /proc/timer_list, /proc/timer_stats, /sys/firmware, and /sys/fs/selinux.**  The default paths that are read only are **/proc/asound, /proc/bus, /proc/fs, /proc/irq, /proc/sys, /proc/sysrq-trigger**, /sys/fs/cgroup.
```

Comment:
shouldn't `/sys/fs/cgroup` be in the ** block?

Reply:
do we need to document that `/sys/fs/cgroup` unmasking will work only on cgroup v2?

Reply:
No, I don't think that is necessary.  This is going to be such a rare occurrence.

Reply:
, /sys/fss/cgroup** is correct

Reply:
Does ** means that you can mount subdirs from that dir? If yes, it is not possible with this one.
You can currently allow rw for the whole /sys/fs/cgroup. You can't choose /sys/fss/cgroup/someOtherDir. This would require much more refactoring in the base function (where  /sys/fs/cgroup are treated in a special way).

Reply:
To complement, please refer to this:
`		cgroupMnt := spec.Mount{

			Destination: "/sys/fs/cgroup",
			Type:        "cgroup",
			Source:      "cgroup",
			Options:     []string{"rprivate", "nosuid", "noexec", "nodev", "relatime", cgroupPerm},
		}
`

in pkg/specgen/generate/oci.go

Reply:
unless ** means access is rw access is granted to all subdirs. Then /sys/fss/cgroup** is correct.

Reply:
If you saw my deleted comments, ignore them. I misinterpreted this block. I will correct this one.

Reply:
corrected now